docs/en/compute/Tensorflow.md

```diff
@@ -23,7 +23,7 @@ on top of Alluxio POSIX API.
 
 * Setup Java for Java 8 Update 60 or higher (8u60+), 64-bit.
 * Alluxio has been set up and is running.
-* [Tensorflow](https://www.tensorflow.org/install/pip) installed. 
+* [Tensorflow](https://www.tensorflow.org/install/pip) installed. This guide uses Tensorflow **v1.15**.
```

Comment:
Is there a command we can provide for users to install v1.15 through pip rather than 2.x+?

Reply:
the tensorflow install page has a section about installing 1.15. I wasn't sure about adding the pip command since the installation guide goes through some setup stuff first.

should i still add it?

here's a snippet from the tensorflow installation doc:

```
Older versions of TensorFlow
For TensorFlow 1.x, CPU and GPU packages are separate:

tensorflow==1.15 —Release for CPU-only
tensorflow-gpu==1.15 —Release with GPU support (Ubuntu and Windows)
```

Reply:
Ah i see. I think we can leave it out then